Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -56,6 +56,7 @@ public extension ChannelPipeline {
     ///         negotiated, or if no protocol was negotiated. Must return a future that completes when the
     ///         pipeline has been fully mutated.
     /// - returns: An `EventLoopFuture<Void>` that completes when the pipeline is ready to negotiate.
+    @available(*, deprecated, message: "Please use Channel.configureHTTP2SecureUpgrade(h2ChannelConfigurator, http1ChannelConfigurator)")
```

Comment:
As discussed with @Lukasa, I'm deprecating `ChannelPipeline.configureHTTP2SecureUpgrade` and replacing it `Channel.configureHTTP2SecureUpgrade`.